cmd/bootkube/recover.go

```diff
@@ -66,6 +70,22 @@ func runCmdRecover(cmd *cobra.Command, args []string) error {
 			return err
 		}
 		backend = recovery.NewEtcdBackend(etcdClient, recoverOpts.etcdPrefix)
+	case recoverOpts.etcdBackupFile != "":
```

Comment:
Maybe just move this above the switch, set the etcdServer, and then let it fall througH?